localstack/utils/common.py

```diff
@@ -491,7 +491,7 @@ def wait_for_port_open(port, http_path=None, expect_success=True, retries=10, sl
         If 'http_path' is set, make a GET request to this path and assert a non-error response. """
     def check():
         if not is_port_open(port, http_path=http_path, expect_success=expect_success):
-            raise Exception()
+            raise Exception(f"Port {port} (path: {http_path or 'no path'}) was not open")
```

Comment:
Thanks for this PR @seveibar ! Just a minor comment (to fix the builds) - can we please use Python 2.x compatible syntax here (as the code in `common.py` gets deployed into some Python 2.x Lambdas as part of the tests):
```
raise Exception('Port %s (path: %s) was not open' % (port, http_path or 'no path'))
```
Thanks!